### DISCUSSION TITLE
Perspective react project not building with latest  version V1.4.1

### DISCUSSION THREAD

**Discussion by Madasamy26** [perspective-dev/perspective#1885] [Q&A] [answered]
### Steps to Reproduce:
 Install dependencies in examples/react sample code in codebase

### Expected Result:
Expecting build without issues 

### Actual Result:
Actually the result is :: 
Failed to load 'C:\Users\Downloads\perspective-react-typescript-example-master\perspective-react-typescript-example-master\webpack.config.js' config
[webpack-cli] C:\Users\Downloads\perspective-react-typescript-example-master\perspective-react-typescript-example-master\node_modules\@finos\perspective\dist\umd\perspective.js:11
var ot=Object.create;var I=Object.defineProperty;var st=Object.getOwnPropertyDescriptor;var at=Object.getOwnPropertyNames;var lt=Object.getPrototypeOf,ut=Object.prototype.hasOwnProperty;var Ne=t=>I(t,"__esModule",{value:!0});var Oe=(t,e)=>()=>(e||t((e={exports:{}}).exports,e),e.exports),ze=(t,e)=>{Ne(t);for(var r in e)I(t,r,{get:e[r],enumerable:!0})},ct=(t,e,r)=>{if(e&&typeof e=="object"||typeof e=="function")for(let n of at(e))!ut.call(t,n)&&n!=="default"&&I(t,n,{get:()=>e[n],enumerable:!(r=st(e,n))||r.enumerable});return t},Ce=t=>ct(Ne(I(t!=null?ot(lt(t)):{},"default",t&&t.__esModule&&"default"in t?{get:()=>t.default,enumerable:!0}:{value:t,enumerable:!0})),t);var Ze=(()=>{for(var t=new Uint8Array(128),e=0;e<64;e++)t[e<26?e+65:e<52?e+71:e<62?e-4:e*4-205]=e;return r=>{for(var n=r.length,i=new Uint8Array((n-(r[n-1]=="=")-(r[n-2]=="="

SyntaxError: Unexpected token .
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (C:\Users\mmariappan\Downloads\perspective-react-typescript-example-master\perspective-react-typescript-example-master\node_modules\@finos\perspective-webpack-plugin\index.js:10:22)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

### Environment:
windoows 10

### Additional Context:
To run perspective in react with webpack any additional config needed?

**Comment by texodus** [comment, marked as answer]
1. Your node.js is too old (package.json has a minimum of [`14.18.2`](https://github.com/finos/perspective/blob/master/package.json#L16)
2. You need to use `yarn start <example>` from the locally-built project root to run these examples, [as per docs](https://perspective.finos.org/docs/md/development.html#build).  They will not build as-written copied out of the monorepo (this is true for most any monorepo)

**Reply by Madasamy26** [reply]
Thanks for your reply

**Reply by Madasamy26** [reply]
I have, fixed the issue
